druntime/test/exceptions/src/invalid_memory_operation.d

```diff
@@ -8,5 +8,6 @@ struct S
 
 void main()
 {
-    new S;
+    foreach(i; 0 .. 100)
+        new S;
```

Comment:
maybe add a `GC.collect();` here to force a collection then?

Reply:
I'm not seeing why we have to force one when it's going to collect at the end of the program anyway. The goal here is to make sure there is at least one piece of garbage that will trigger the error, and making 100 of them means at least one of those won't be referred to in stack/registers.

Probably could be 2, but 100 is cheap...